spec/order_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'order.rb'
+
+describe Order do
+  subject(:order) { described_class.new({ 'beef-chuck brisket sub' => 4.50, 'pastrami salami sub' => 5.00 }) }
```

Comment:
Metrics/LineLength: Line is too long. [110/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.